lib/mutations/command.rb

```diff
@@ -56,7 +56,7 @@ def input_filters
     # Instance methods
     def initialize(*args)
       @raw_inputs = args.inject({}.with_indifferent_access) do |h, arg|
-        raise ArgumentError.new("All arguments must be hashes") unless arg.is_a?(Hash)
+        raise ArgumentError.new("All arguments must be hashes or respond to to_hash") unless arg.respond_to?(:to_hash)
```

Comment:
I don't think we should change this error message. When the error is raised, the right response is almost always to pass a hash instead, rather than implementing `to_hash` for whatever object was just passed. Mentioning `to_hash` is more likely to guide users towards doing the wrong thing, IMO.

Reply:
Makes sense